ivy_tests/test_ivy/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .. import config
-
-if hasattr(config, "try_except"):
-    config.try_except()
```

Comment:
Can we revert unrelated changes?

Reply:
This is because of some corner case where `config["ivyh"]` is undefined as the key `ivyh` isn't present there, might be a mp thing, but not sure, the replaced code handles this corner case too.